docs/src/main/asciidoc/spring-cloud-stream-support.adoc

```diff
@@ -531,6 +531,112 @@ When the batch-consumer mode is enabled, the specific headers of batched message
 
 NOTE: When publish messages, all the above batch headers if exist will be removed from the messages to send.
 
+===== Multi Binders Support
+Step 1. To use multiBinders of EventHubs, we need to configure below properties in application.yml
+[source,yaml]
+----
+spring:
+  cloud:
+    stream:
+      function:
+        definition: consume1;supply1;consume2;supply2
+      bindings:
+        consume1-in-0:
+          destination: ${EVENTHUB_NAME_01}
+          group: ${CONSUMER_GROUP_01}
+        supply1-out-0:
+          destination: ${THE_SAME_EVENTHUB_NAME_01_AS_ABOVE}
+        consume2-in-0:
+          binder: eventhub-2
+          destination: ${EVENTHUB_NAME_02}
+          group: ${CONSUMER_GROUP_02}
+        supply2-out-0:
+          binder: eventhub-2
+          destination: ${THE_SAME_EVENTHUB_NAME_02_AS_ABOVE}
+      binders:
+        eventhub-1:
+          type: eventhubs
+          default-candidate: true
+          environment:
+            spring:
+              cloud:
+                azure:
+                  eventhubs:
+                    namespace: ${EVENTHUB_NAMESPACE_01}
```

Comment:
here the property should be connection-string
![image](https://user-images.githubusercontent.com/63028776/154920695-d45e6833-1444-4363-b18f-84d036ee1f23.png)